docs/installation/howto/optional.rst

```diff
@@ -40,11 +40,7 @@ The following optional dependencies are supported:
 
 * **serial** - the `pyserial`_ package to work with serial data.
 
-* **all_non_platform** - installs **tls**, **conch**, **soap**, and **serial** options.
-
-* **macos_platform** - **all_non_platform** options and `pyobjc`_ to work with Objective-C apis.
-
-* **windows_platform** - **all_non_platform** options and `pywin32`_ to work with Windows's apis.
+* **all** - installs **tls**, **conch**, **soap**, and **serial** options.
```

Comment:
there's a bunch of `* `TwistedChecker`_, only available on python2` that I'm going to gently suggest you also fix

Reply:
I overhauled that page with updated links and package listing.  Should there be a news fragment for it?